src/gameconfig.cpp

```diff
@@ -256,7 +258,7 @@ byte *CGameConfig::HexToByte(const char *src, size_t &length)
 	}
 
 	length = strlen(src) / 4;
-	uint8_t *dest = new uint8_t[length];
+	uint8_t *dest = new uint8_t[length + 1];
```

Comment:
Is the final null byte initialized here? I don’t know much about C++ initialization so just making sure.